src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -353,26 +355,25 @@ private void exportDescription(final URI uri) throws FcrepoOperationFailedExcept
 
     }
 
-    private List<URI> filterInboundReferences(final URI uri, final Model model) {
-        final List<URI> inboundMembers = new ArrayList<>();
+    private Set<URI> filterInboundReferences(final URI uri, final Model model) {
+        final String withSlash = withSlash(uri).toString();
+        final String withoutSlash = withoutSlash(uri).toString();
+        final Set<URI> inboundMembers = new HashSet<>();
         final List<Statement> removeList = new ArrayList<>();
-        for (final String p : config.getPredicates()) {
-            final StmtIterator inbound = model.listStatements(null, createProperty(p), (RDFNode)null);
-            while (inbound.hasNext()) {
-                final Statement s = inbound.next();
-                final String subject = s.getSubject().toString();
-                if (!subject.startsWith(uri.toString())) {
-                    removeList.add(s);
-                    inboundMembers.add(URI.create(subject));
-                }
+        for (final StmtIterator inbound = model.listStatements(); inbound.hasNext(); ) {
+            final Statement s = inbound.next();
+            final String subject = s.getSubject().toString();
+            if (!subject.equals(withSlash) && !subject.equals(withoutSlash)) {
+                removeList.add(s);
+                inboundMembers.add(URI.create(subject));
```

Comment:
Yes, if there are many children of a single node, then they might all call `exportDescription` — but there is a pretty quick check at the beginning to avoid re-fetching: https://github.com/fcrepo4-labs/fcrepo-import-export/blob/master/src/main/java/org/fcrepo/importexport/exporter/Exporter.java#L302

I think it could be worth checking whether managing a global list of URIs that have already been exported, but we'd need to make sure that didn't get too big or it could be a memory problem for larger exports.